File: program/find-surface-area-of-a-cuboid/find_surface_area_of_a_cuboid.dart
import 'dart:io';
void main(){
  print("Enter the length :");
  var length =int.parse(stdin.readLineSync());
  print("Enter the width :");
  var width =int.parse(stdin.readLineSync());
  print("Enter the height :");
  var height =int.parse(stdin.readLineSync());
  var surface_area = 2 * ((length * width) + (length * height) + (width * height));
  print('Surface area of cuboid : $surface_area');
}

File: program/find-surface-area-of-a-cuboid/find_surface_area_of_a_cuboid.rs
fn main() {
    let length = 2.0;
    let width = 3.0;
    let height = 4.0;
    let surfaceArea = 2.0 * (length * width + length * height + width * height);

    println!("Surface area of cuboid is {0}", surfaceArea as f32);
}
